templates/repo/actions/workflow_dispatch.tmpl

```diff
@@ -0,0 +1,76 @@
+<script type="module">
+	$('#runWorkflowDispatchForm').form({
+		fields: {
+			required_input : 'empty'
+		}
+	});
+</script>
+
+<div class="ui ignored info message">
+	This workflow has a workflow_dispatch event trigger.
+	<button class="ui right floated mini button" onclick="$('#runWorkflowDispatchModal').modal('show');">{{ctx.Locale.Tr "actions.workflow.run"}}{{svg "octicon-triangle-down" 14 "dropdown icon"}}</button>
+</div>
+<div id="runWorkflowDispatchModal" class="ui mini modal">
+	<div class="content">
+		<form id="runWorkflowDispatchForm" class="ui form" action="{{$.Link}}/run?workflow={{$.CurWorkflow}}&actor={{$.CurActor}}&status={{.Status}}" method="post">
+			{{.CsrfTokenHtml}}
+			<div class="ui inline field required">
+				<label>{{ctx.Locale.Tr "actions.workflow.from_ref"}} :</label>
+				<div class="ui floating dropdown icon button">
+					<input type="hidden" name="ref" value="refs/heads/{{index .Branches 0}}">
+					<i>{{svg "octicon-git-branch"}}</i>
+					<span class="text">{{index .Branches 0}}</span>
+					<div class="menu">
+						<div class="ui icon search input">
+							<i class="search icon"></i>
+							<input type="text" placeholder="{{ctx.Locale.Tr "repo.filter_branch_and_tag"}}">
+						</div>
+						<div class="scrolling menu">
+							<div class="header">
+								{{svg "octicon-git-branch"}}
+								Branches
+							</div>
+							{{range $key, $item := .Branches}}
+							<div class="item {{if eq $key 0}}active selected{{end}}" data-value="refs/heads/{{.}}">{{.}}</div>
+							{{end}}
+							<div class="divider"></div>
+							<div class="header">
+								{{svg "octicon-tag"}}
+								Tags
+							</div>
+							{{range $key, $item := .Tags}}
+							<div class="item" data-value="refs/tags/{{.}}">{{.}}</div>
+							{{end}}
+						</div>
+					</div>
+				</div>
+			</div>
+
+			<div class="divider"></div>
+			{{if .WorkflowDispatchConfig}}
+			{{range $key, $item := .WorkflowDispatchConfig.Inputs}}
+			<div class="ui field {{if .Required}}required{{end}}">
```

Comment:
![image](https://github.com/go-gitea/gitea/assets/18380374/abaa3036-9705-480d-af2a-48596164cb4e)
boolean have two labels, if it is required, you can see two `*`

Reply:
Fixed. Checkbox has its own label